helm/charts/templates/cluster_role_binding.yaml

```
@@ -0,0 +1,19 @@
+{{ if not .Values.namespaced }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "platform-health.fullname" . }}
+  {{- $roleBinding := .Values.roleBinding | default dict }}
+  {{- $labels := mergeOverwrite (.Values.common_labels | default dict) ($roleBinding.labels | default dict) | toYaml | nindent 4 }}
+  labels: {{ $labels }}
+  {{- $annotations := mergeOverwrite (.Values.common_annotations | default dict) ($roleBinding.annotations | default dict) | toYaml | nindent 4 }}
+  annotations: {{ $annotations }}
+subjects:
+  - kind: ServiceAccount
+    name: platform-health
```

```suggestion
    name: [platform-health
](name: {{ template "platform-health.fullname" . }})
```
… I think? It's no longer hard-coded.